### ISSUE TITLE
please install pure js modules in /usr/share/nodejs

### ISSUE THREAD

**Issue by kapouer** [LeoIannacone/npm2deb#145]
this is supported since nodejs 10.12, so won't be a mess to backport.

```
  * Multiarch and DFHS:
    + patch adding --node-relative-path configuration variable
    + "/usr/lib/<$DEB_HOST_MULTIARCH>/nodejs" for c++ addons
    + "/usr/share/nodejs" for pure javascript modules
    + "/usr/lib/nodejs" for smooth migration of current modules
    + Expose those configuration variables in
      process.config.variables.node_relative_path
      process.config.variables.arch_triplet
    + In particular, if prefix is /usr/local all paths move there.
    + Fix test expecting lib/node, should be lib/nodejs.
    (Closes: #901474)
```

**Comment by guimard** [comment]
Then we should modify pkg-js-tools dependency to require version 0.7 at least _(see [#931675](https://bugs.debian.org/931675))_

**Comment by kapouer** [comment]
An addon would typically have "gyp" or "nan" (or both) in `devDependencies` or in `dependencies`'s package.json.

**Comment by guimard** [comment]
Fixes by #147
